plugin/trino-exchange/src/main/java/io/trino/plugin/exchange/s3/ExchangeS3Config.java

```diff
@@ -0,0 +1,163 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.exchange.s3;
+
+import io.airlift.configuration.Config;
+import io.airlift.configuration.ConfigDescription;
+import io.airlift.configuration.ConfigSecuritySensitive;
+import io.airlift.units.DataSize;
+import io.airlift.units.Duration;
+import io.airlift.units.MaxDataSize;
+import io.airlift.units.MinDataSize;
+import io.airlift.units.MinDuration;
+
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+
+import java.util.concurrent.TimeUnit;
+
+import static io.airlift.units.DataSize.Unit.MEGABYTE;
+
+public class ExchangeS3Config
+{
+    private String s3AwsAccessKey;
+    private String s3AwsSecretKey;
+    private String s3Endpoint;
+    private int s3MaxErrorRetries = 10;
+    private Duration s3ConnectTimeout = new Duration(5, TimeUnit.SECONDS);
+    private Duration s3SocketTimeout = new Duration(5, TimeUnit.SECONDS);
+    private int s3MaxConnections = 500;
+    private boolean pinS3ClientToCurrentRegion;
+    private DataSize s3StreamingPartSize = DataSize.of(5, MEGABYTE);
```

Comment:
nit: maybe drop `s3` prefix from field names (can stay in getters/setters).

Reply:
I'm trying to be consistent with `HiveS3Config.java`